pkg/roachpb/span_config.proto

```diff
@@ -166,56 +166,99 @@ message SpanConfig {
   bool rangefeed_enabled = 10;
 }
 
+// SystemSpanConfigTarget specifies the target of system span configurations.
+message SystemSpanConfigTarget {
+  option (gogoproto.equal) = true;
+
+	// SourceTenantID is the ID of the tenant that specified the system span
+	// configuration.
+	TenantID SourceTenantID = 1 [(gogoproto.nullable) = false];
+
+	// TargetTenantID is the ID of the tenant that the associated system span
+	// configuration applies to. If the host tenant is the source then the
```

Comment:
nit: Can we reword the sentence "If the host tenant is the source then the system span config applies over all ranges in the system (including those belonging to secondary tenants)." to mention that both SourceTenantID and TargetTenantID have to be set to the host tenant ID.